index.html

```diff
@@ -174,12 +175,32 @@ <h2>About</h2>
             <li>GitLab</li>
             <li>Terminal</li>
           </ul>
-          <img src="images/ic_skills.svg" class="svg-image3 position" alt="svg image">
+          <img src="images/ic_skills.svg" class="about-images" alt="svg image">
         </div>
       </div>
     </div>
   </section>
 
+  <form action="https://formspree.io/f/xwkaywbg" class="contact-form d-flex flex-cols" method="POST">
+    <h2 class="typography primary-color">Get Started</h2>
+    <div class="cta-summary poppins"><p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.
+    </p>
+  </div>
+    <span class="d-flex flex-cols input-wrapper margin-tp">
+      <label for="name" class="poppins">Your name</label>
+      <input class="primary-color no-outline" type="text" name="name" id="name" placeholder="" title="Your name" maxlength="30" required>
+    </span>
+    <span class="d-flex flex-cols input-wrapper">
+      <label for="email" class="poppins">Your email</label>
+      <input class="primary-color no-outline" type="text" name="email" id="email" placeholder="" title="Your email" required>
+    </span>
+    <textarea class="input-wrapper no-outline primary-color" name="message" id="message" cols="30" rows="10" placeholder="Enter your message here..." maxlength="500" required></textarea>
+    <span class="form-submit poppins dark-bg">
+      <button type="submit" class="dark-bg">Start collaboration</button>
+    </span>
```

Comment:
- [ ] The Start collaboration button should follow the button styling: Active, Hover and Pressed (see design)